packages/template-phpstan-compiler/src/Reporting/TemplateErrorsFactory.php

```diff
@@ -68,4 +76,15 @@ private function resolveNearestPhpLine(array $phpToTemplateLines, int $desiredLi
 
         return $lastTemplateLine;
     }
+
+    private function resolveTipForMessage(string $message): ErrorMessageWithTip
+    {
+        $tip = '';
+        $undefinedFilter = str_replace('Access to an undefined property Latte\Runtime\FilterExecutor::$', '', $message);
+        if ($undefinedFilter !== $message) {
+            $message = 'Undefined filter ' . $undefinedFilter;
```

Comment:
```suggestion
            $message = 'Undefined filter "' . $undefinedFilter . ''";
```

Quotes in the messge help to highlight the specific value from generic message

Reply:
OK I agree, I'll add it